components/twitter/user-tweets.js

```diff
@@ -27,37 +25,29 @@ module.exports = {
   }, 
   methods: {},
   async run(event) {
-    const from = `from:${this.from.replace('@','')}`
+    const screen_name = this.screen_name //.replace('@','')
     const since_id = this.db.get("since_id")
-    const { lang, locale, geocode, result_type, enrichTweets, includeReplies, includeRetweets, maxRequests, count } = this
-    let q = from, max_id, limitFirstPage
-    
-    // join "from" filter and search keywords
-    if (this.q) {
-      q += ` ${this.q}`
-    }
-    
-    if (!since_id) {
-      limitFirstPage = true
+    const { enrichTweets, includeReplies, includeRetweets, count } = this
+    let max_id
+
+    if(!includeReplies) {
+      exclude_replies = true
     } else {
-      limitFirstPage = false
+      exclude_replies = false
     }
+    include_rts = includeRetweets
```

Comment:
Any reason you can't use the `includeRetweets` var, instead of re-defining?

Reply:
done